src/PrestaShopBundle/Form/Admin/Product/ProductInformation.php

```diff
@@ -222,6 +222,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'allow_delete' => true,
             ])
             ->add('id_manufacturer', FormType\ChoiceType::class, [
+                'placeholder' => false,
```

Comment:
I have tested under branch 1.7.7.x and it successfully removes the empty option in select brand of product page. However, when I create new product, the first brand in list is now automatically applied to product. That empty option is (maybe) what tells Prestashop to not store any brand for the product until the merchant decides it. What do you think of using the `attr` to add a class (with some CSS properties that only hide and not remove option from DOM)

Reply:
That's too tricky for me, maybe we can manage to get an option displaying "Select brand" which is empty ? That would be better than an empty option and avoid tricky css things

Reply:
@NeOMakinG , you are right, your option is better than CSS "hiding".